src/parser.rs

```diff
@@ -549,6 +579,48 @@ fn parse_import(
     (ParsedNamespace::new(), error)
 }
 
+fn parse_name(tokens: &[Token], index: &mut usize) -> (String, Span, Option<JaktError>) {
+    if let Some(token) = tokens.get(*index) {
+        match &token.contents {
+            TokenContents::Name(name) => {
+                *index += 1;
+                (name.to_string(), token.span, None)
+            }
+            _ => (
+                String::new(),
+                tokens[*index - 1].span,
+                Some(JaktError::ParserError(
+                    format!("expecting name"),
```

Comment:
The helper function here seems fine if we're using it a lot, but it might be nice to have more descriptive errors.